mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,154 @@
+
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/create-group-default.constants';
+import { styles } from './styles';
+
+const UAMConfigureGroup: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const { group } = useAppSelector((state) => state.uamGroupEdit);
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const { page: usersPage, handlePageChange: handleUserPageChange } =
+    usePagination();
+
+  const {
+    page: permissionsPage,
+    handlePageChange: handlePermissionsPageChange,
+  } = usePagination();
+
+  const { control, handleSubmit, errors, reset, watch } =
+    useAppForm<GroupsUpdateRequestDto>({
+      defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+      validationSchema: groupCreateClient,
+    });
+
+  const {
+    items: permissionIds,
+    handleToggle: handleTogglePermissions,
+    setItems: setDefaultPermissionIds,
+  } = useSelectedItems<number>(group?.permissionIds ?? []);
+
+  const {
+    items: userIds,
+    handleToggle: handleToggleUsers,
+    setItems: setDefaultUserIds,
+  } = useSelectedItems<number>(group?.userIds ?? []);
+
+  const paginationForUsersTable = {
+    page: usersPage,
+    setPage: handleUserPageChange,
+  };
+
+  const paginationForPermissionsTable = {
+    page: permissionsPage,
+    setPage: handlePermissionsPageChange,
+  };
+
+  const groupPayload: GroupsUpdateRequestDto = {
+    name: watch.name,
+    permissionIds,
+    userIds,
+  };
+
+  const handleCreateOrEditGroup = async (): Promise<void> => {
+    if (!group) {
+      await dispatch(groupsCreationActions.createGroup(groupPayload)).unwrap();
+    } else {
+      await dispatch(
+        uamGroupEditActions.editGroup({
+          id: group.id,
+          payload: groupPayload,
+        }),
+      ).unwrap();
+    }
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  const handleCancelEdit = async (): Promise<void> => {
+    dispatch(uamGroupEditActions.cancelEdit);
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  useEffect(() => {
+    dispatch(groupsCreationActions.getPermissions());
+  }, []);
+
+  useEffect(() => {
+    dispatch(
+      groupsCreationActions.getUsersForCreation({
+        page: usersPage,
+        count: PaginationDefaultValue.DEFAULT_COUNT,
+      }),
+    );
+  }, [usersPage]);
+
+  useEffect(() => {
+    if (group) {
+      setDefaultPermissionIds(group.permissionIds);
+      setDefaultUserIds(group.userIds);
+      reset({ name: group.name });
+    }
+  }, [group]);
+
+  return (
+    <View style={styles.container}>
+      <ScrollView>
+        <View style={styles.inputContainer}>
+          <Input
+            name="name"
+            placeholder="Enter group name"
+            label="Group name"
+            control={control}
+            errors={errors}
+          />
+        </View>
+
+        <Text style={styles.title}>Add users to the Group - Optional</Text>
+        <UsersTable
```

Comment:
Tables should look now same as on uam screen, please attach new screenshot to pr

Reply:
Do we need actions column with delete functionallity for users table in Group edit screen?

Reply:
No, we don't need action buttons on this screen